deployability/modules/testing/tests/helpers/central.py

```diff
@@ -19,23 +19,33 @@ def install_aio(inventory_path, wazuh_version) -> None:
             wazuh_version (str): major.minor.patch
 
         """
-        wazuh_version = '.'.join(wazuh_version.split('.')[:2])
         os_name = HostInformation.get_os_name_from_inventory(inventory_path)
 
+        if live == "False":
+            s3_url = 'packages-dev.wazuh.com'
+        else:
+            s3_url = 'packages.wazuh.com'
+
+        release = '.'.join(wazuh_version.split('.')[:2])
+
+
+        logger.info(f'Installing Manager with https://{s3_url}/{release}/wazuh-install.sh')
+
         if HostInformation.has_curl(inventory_path):
             commands = [
-                f"curl -sO https://packages.wazuh.com/{wazuh_version}/wazuh-install.sh && sudo bash ./wazuh-install.sh -a --ignore-check"
+                f"curl -sO https://{s3_url}/{release}/wazuh-install.sh",
+                f"sudo bash ./wazuh-install.sh -a --ignore-check"
             ]
         else:
             commands = [
-                f"wget https://packages.wazuh.com/{wazuh_version}/wazuh-install.sh && sudo bash ./wazuh-install.sh -a --ignore-check"
+                f"wget https://{s3_url}/{release}/wazuh-install.sh",
+                f"sudo bash ./wazuh-install.sh -a --ignore-check"
```

Comment:
Revert to a single line

Reply:
Done in [a7aa606](https://github.com/wazuh/wazuh-qa/pull/5392/commits/a7aa6062b7d70d81d64975f698f29e7d4959c353)